### ISSUE TITLE
background script is unresponsive

### ISSUE THREAD

**Issue by kshitij10496** [kshitij10496/read-later#5]
The background script : `background.js` is not responding to the user's click on the extension icon.

**Comment by singh1114** [comment]
Yes it does..... I replaced some code and printed it on the console. Indeed it got printed may be their is some error in your function.

chrome.browserAction.onClicked.addListener( function(tab) {
  console.log("singh");
});

**Comment by vijeth-aradhya** [comment]
I'm also learning about chrome extensions. I don't know where I'm going wrong, but can someone please help me out here? @singh1114 

When I console log the chrome object (after stringifying it), I couldn't find browserAction at all

![chrome](https://cloud.githubusercontent.com/assets/15227565/19022202/1fff1c68-88f0-11e6-97b6-8f3d782b6691.png)

**Comment by singh1114** [comment]
http://stackoverflow.com/questions/10081898/how-to-debug-google-chrome-background-script

**Comment by vijeth-aradhya** [comment]
@singh1114 Oh alright thanks :)

@kshitij10496 The function is being called, but there's an error in your "printBookmarks" function like @singh1114 pointed out

**Comment by vijeth-aradhya** [comment]
@kshitij10496 Instead of `console.log(bookmark.title)` and checking, why don't you alert and see it.
I could see the bookmark title that way

**Comment by singh1114** [comment]
@kshitij10496 yes working for me too.

**Comment by singh1114** [comment]
@kshitij10496 The value is also getting stored in the variable. so don't worry and accomplish whatever you wanted to do.

**Comment by akshatmaheshwari** [comment]
@kshitij10496 It works fine for me also. And you can remove the reference to `background.js` from `test.html`.
